### ISSUE TITLE
Adding a whale wallet breaks iOS

### ISSUE THREAD

**Issue by AW-STJ** [AlphaWallet/alpha-wallet-ios#2594]
Not a critical issue, but something to work on when optimising app's performance.

The app becomes very slow once a whale account is added as a watch address. 

Steps to reproduce:
1. Go to watch wallet
2. Add 0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2
3. App would be slow for now
4. Change language in settings - app crashes in my iPad mini. iPhone works better, but app performance is very poor

I have already shared my crash reports, and have noted my name in the logs. Not sure whether useful or not. But something to look into

**Comment by hboon** [comment]
Yes. Some performance related work is currently being worked on. E.g. #2374
